docker/Dockerfile

```diff
@@ -71,6 +67,10 @@ RUN python3 -m pip install --no-cache-dir --upgrade \
     pycodestyle \
     pydocstyle \
     pyflakes \
+    pylint \
+    pylint-django \
+    --ignore-installed pyyaml \
```

Comment:
This is because of a conflict with an OS-installed version.  It makes me wonder if we should just switch the image to using a venv for Python stuff.

Reply:
I like the idea of a Python venv in the Docker image. It's also nice to show in a Docker image how we install and execute things locally.

Reply:
i think using a venv in the docker image makes a lot of sense.